# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/interfaces/nsIAccessibleEvent.idl
# Commit: 16b49e8c315b
# Full Hash: 16b49e8c315bb512c0bc406670d3b4be653185b4
# Author: James Teh <jteh@mozilla.com>
# Date: 2023-11-30 04:12:04
# Regressor Bug: 1862802
# File Overlap Count: 1
# Description:
#   Bug 1862802 part 5: Remove virtual cursor change event. r=eeejay
#   
#   Depends on D192645
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D192646
# ==============================================================================

diff -r 5a6669e44c53 -r 16b49e8c315b accessible/interfaces/nsIAccessibleEvent.idl
--- a/accessible/interfaces/nsIAccessibleEvent.idl	Wed Nov 29 03:53:20 2023 +0000
+++ b/accessible/interfaces/nsIAccessibleEvent.idl	Wed Nov 29 03:53:20 2023 +0000
@@ -183,45 +183,40 @@
   const unsigned long EVENT_OBJECT_ATTRIBUTE_CHANGED = 0x0022;
 
   /**
-   * A cursorable's virtual cursor has changed.
-   */
-  const unsigned long EVENT_VIRTUALCURSOR_CHANGED = 0x0023;
-
-  /**
    * An object's text Value has changed.
    */
-  const unsigned long EVENT_TEXT_VALUE_CHANGE = 0x0024;
+  const unsigned long EVENT_TEXT_VALUE_CHANGE = 0x0023;
 
   /**
    * An accessible's viewport is scrolling.
    */
-  const unsigned long EVENT_SCROLLING = 0x0025;
+  const unsigned long EVENT_SCROLLING = 0x0024;
 
   /**
    * An accessible is making an explicit announcement.
    */
-  const unsigned long EVENT_ANNOUNCEMENT = 0x0026;
+  const unsigned long EVENT_ANNOUNCEMENT = 0x0025;
 
   /**
    * A live region has been introduced. Mac only.
    */
-  const unsigned long EVENT_LIVE_REGION_ADDED = 0x0027;
+  const unsigned long EVENT_LIVE_REGION_ADDED = 0x0026;
 
   /**
    * A live region has been removed (aria-live attribute changed). Mac Only.
    */
-  const unsigned long EVENT_LIVE_REGION_REMOVED = 0x0028;
+  const unsigned long EVENT_LIVE_REGION_REMOVED = 0x0027;
 
   /**
    * A reorder event that has been coalesced into a mutation
    * of an ancestor's subtree.
    */
-  const unsigned long EVENT_INNER_REORDER = 0x0029;
+  const unsigned long EVENT_INNER_REORDER = 0x0028;
 
   /**
    * Help make sure event map does not get out-of-line.
    */
-  const unsigned long EVENT_LAST_ENTRY = 0x002A;
+  const unsigned long EVENT_LAST_ENTRY = 0x0029;
 
   /**
    * The type of event, based on the enumerated event values